server/public/model/system.go

```diff
@@ -119,6 +121,9 @@ type SupportPacket struct {
 	LicenseTo             string `yaml:"license_to"`
 	LicenseSupportedUsers int    `yaml:"license_supported_users,omitempty"`
 	LicenseIsTrial        string `yaml:"license_is_trial,omitempty"`
+	CustomerID            string `yaml:"customer_id"`
+	LicenseID             string `yaml:"license_id"`
+	ServerID              string `yaml:"server_id"`
```

Comment:
It doesn't quite fit in the License category. Maybe it should be placed after `Build information`.